srcpkgs/icecream/template

```diff
@@ -0,0 +1,49 @@
+# TODO
+# * settle on naming (icecream vs. icecc).  https://repology.org/project/icecream/versions
+# * system user obviously cannot chroot, which makes building in custom envs impossible. Add a second service `iceccd-root` that can?
```

Comment:
iceccd does only drop the priv after setting chroot capabilities, so the way you do it in the current service should already work?

(I only quickly read the code, not actually test it)

Reply:
that's probably a leftover from their old test suite which complained about it